docs/indepth/installation.rst

```diff
@@ -173,49 +175,49 @@ Using Amazon Web Services
 
 #. Add the *Key* to your local key chain (download it from the AWS web interface first if necessary):
 
-   .. code-block:: sh
+   .. prompt:: bash laptop$
 
-      yourlaptop$ mv ~/Downloads/my-instance.pem ~/.ssh/
-      yourlaptop$ chmod 600 ~/.ssh/my-instance.pem
-      yourlaptop$ ssh-add ~/.ssh/my-instance.pem
+      mv ~/Downloads/my-instance.pem ~/.ssh/
+      chmod 600 ~/.ssh/my-instance.pem
+      ssh-add ~/.ssh/my-instance.pem
 
-#. Look up the public DNS name or public IP address of the new instance and log in as user "fedora", e.g.:
+#. Look up the public DNS name or public IP address of the new instance and, depending on the OS, log in as user "fedora", "centos", or "ubuntu" e.g.:
```

Comment:
Same as the comment about DO - I think that this is useful work to keep for when we support non-Fedora on cloud providers, but that is not now.